src/package.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8" ?>
 <package xmlns="http://www.mendix.com/package/1.0/">
-	<clientModule name="Dynamic Image" version="0.1" xmlns="http://www.mendix.com/clientModule/1.0/">
+	<clientModule name="Dynamic Image" version="1.0.1" xmlns="http://www.mendix.com/clientModule/1.0/">

```

Comment:
`1.0.0`
Think wou should add a comment to this package XML
``` xml
<!-- Custom upgraded to Mendix 7, based on version 0.1-->
```
Similar comment should be on top of every file that is changed.

Reply:
Added